recipes/la_forge/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "la_forge" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0b457003af2bcfe49b3320708520eb9e93a7b882f6b318b388f3107b9f630eee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.16
+    - scipy >=1.0.0
+    - matplotlib-base
```

```suggestion
    - matplotlib-base >=2.0.0
```

https://github.com/nanograv/la_forge/blob/e167b0c17b8721e5e383d0fc96ca175e0f3e0c78/setup.py#L16